src/v/cluster/security_frontend.cc

```diff
@@ -323,23 +323,35 @@ security_frontend::get_bootstrap_user_creds_from_env() {
     }
 
     ss::sstring creds_str = creds_str_ptr;
-    auto colon = creds_str.find(":");
-    if (colon == ss::sstring::npos || colon == creds_str.size() - 1) {
+    std::string_view creds = creds_str_ptr;
+    std::vector<std::string_view> parts;
+    parts.reserve(3);
+    boost::algorithm::split(parts, creds, [](char c) { return c == ':'; });
```

Comment:
This probably requires an include

Reply:
I assume the proper include for this would be:
```
#include <boost/algorithm/string/split.hpp>
```